setup.py

```diff
@@ -82,6 +82,33 @@ def get_data_files():
                     glob.glob('examples/wrappers/*.sh'))]
 
     data_files.append((get_avocado_libexec_dir(), glob.glob('libexec/*')))
+    data_files += [(get_dir(['usr', 'share', 'avocado', 'gdb-prerun-scripts'],
```

Comment:
I don't really mind using in-place add for lists. It's probably faster than calling the `extend()`. What troubles me is this alternation between `append()` and `+=`, depending on the number of items to be included into the list. For consistence, we should use `data_files.append(item1)` and `data_files.extend([item2, item3])` OR `data_files += [item1]` and `data_files += [item2, item3]`.
Not your patch fault anyway. Your patch is like the solar eclipse: it looks good while hurting my eyes.

![Eclipse](http://www.telegraph.co.uk/content/dam/news/2017/08/21/TELEMMGLPICT000137989572_trans_NvBQzQNjv4BqpVlberWd9EgFPZtcLiMQfyf2A9a6I9YchsjMeADBa08.jpeg?imwidth=450)